plugins/config-cleanup.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: config-cleanup
+spec:
+  version: "v0.2.0"
+  shortDescription: Automatically clean up your kubeconfig
```

Comment:
Can you add a long description field?

Why is it automatic? How does it clean up? What parts does it cleanup?

Ideally these should be answered without installing the plugin.

Reply:
I've updated the manifest with a longDescription